packages/components/src/Chip/Chip.test.tsx

```diff
@@ -65,12 +59,6 @@ describe("Chip", () => {
 
     expect(getByRole("button")).toHaveAttribute("aria-label", "Test Label");
   });
-
```

Comment:
Chip no longer defaults the aria-label to the passed in label, so this test is no longer valid (was causing the double up errors in testing because two labels were present on the screen)